apps/glusterfs/volume_entry.go

```diff
@@ -420,11 +420,35 @@ func (v *VolumeEntry) cleanupCreateVolume(db wdb.DB,
 
```

Comment:
Your commit message seems to indicate that this is block volume specific. However, this does not appear to be the case in the code. Can you ether explain to me why it is block volume specific or update your commit.

Reply:
I was testing only block volume creation failure scenario, update commit message as this related to volume creation failure.

Reply:
updated commit message, Thanks